showcase/src/app/app.component.html

```diff
@@ -17,7 +17,7 @@
 
 			<div class="stark-app-bar-content-center">
 				<div class="app-logo">
-					<p>App Logo</p>
+					<p><stark-app-logo></stark-app-logo></p>
```

Comment:
Now that we use the `AppLogo` component, does it really need to be inside a `<p>` element? I find it strange...